promscale/sql-api.md

```diff
@@ -17,23 +17,37 @@ WHERE n.nspname OPERATOR(pg_catalog.~) '^(prom)$' COLLATE pg_catalog.default
- jsonb                         | labels label_array                                       | jsonb            | jsonb converts a labels array to a JSONB object.
- key_value_array               | labels label_array, OUT keys text[], OUT vals text[]     | record           | key_value_array converts a labels array to two arrays: one for keys and another for values.
- matcher                       | labels jsonb                                             | matcher_positive | matcher returns a matcher for the JSONB, __name__ is ignored. The matcher can be used to match against a label array using @> or ? operators.
- register_metric_view          | schema_name text, view_name text, if_not_exists boolean  | boolean          | Register metric view with Promscale. This will enable you to query the data with PromQL and set data retention policies through Promscale. Schema name and view name should be set to the desired schema and view you want to use. Note: underlying view needs to be based on an existing metric in Promscale (should use its table in the FROM clause). 
- reset_metric_chunk_interval   | metric_name text                                         | boolean          | reset_metric_chunk_interval resets the chunk interval for a specific metric to using the default.
- reset_metric_retention_period | metric_name text                                         | boolean          | reset_metric_retention_period resets the retention period for a specific metric to using the default.
- set_default_chunk_interval    | chunk_interval interval                                  | boolean          | set_default_chunk_interval set the chunk interval for any metrics (existing and new) without an explicit override.
- set_default_retention_period  | retention_period interval                                | boolean          | set_default_retention_period set the retention period for any metrics (existing and new) without an explicit override.
- set_metric_chunk_interval     | metric_name text, chunk_interval interval                | boolean          | set_metric_chunk_interval set a chunk interval for a specific metric (this overrides the default).
- set_metric_retention_period   | metric_name text, new_retention_period interval          | boolean          | set_metric_retention_period set a retention period for a specific metric (this overrides the default).
- val                           | label_id integer                                         | text             | val returns the label value from a label id.
- unregister_metric_view        | schema_name text, view_name text, if_not_exists boolean  | boolean          | Unregister metric view with Promscale. Schema name and view name should be set to the metric view already registered in Promscale. 
\ No newline at end of file
+ |Name|Arguments|Type|Description|
+|-|:-:|:-:|:-|
```

Comment:
The colons aren't being honoured here. I think maybe Gatsby ignores them?

Reply:
Yes!!! Looks like Gatsby ignores them. I will redo the table may be.